custom_components/waste_collection_schedule/waste_collection_schedule/source/bexley_gov_uk.py

```diff
@@ -34,16 +34,8 @@ def __init__(self, uprn):
         self._uprn = str(uprn).zfill(12)
 
     def fetch(self):
-
         s = requests.Session()
 
-        # Set up session
-        timestamp = time_ns() // 1_000_000  # epoch time in milliseconds
-        session_request = s.get(
-            f"https://mybexley.bexley.gov.uk/apibroker/domain/mybexley.bexley.gov.uk?_={timestamp}",
-            headers=HEADERS,
-        )
```

Comment:
This might have implication on the functionality of the source, as the request may add cookies to the session. So double-check the integration still works or just remove the assignment and leave the request as is

Reply:
Good point - I did run the full test suite before and after these changes, and didn't see any unintended change.

Reply:
```
python test_sources.py -s bexley_gov_uk -i -l -t
Testing source bexley_gov_uk ...
  found 5 entries for Test_001
    2024-05-09 : Food 23 ltr Caddy [mdi:food]
    2024-05-09 : Plastic 55 ltr Box [mdi:recycle]
    2024-05-09 : Paper & Cardboard & 55 ltr Box [mdi:newspaper]
    2024-05-09 : Glass 55 ltr Box [mdi:glass-fragile]
    2024-05-15 : Residual 180 ltr bin [mdi:trash-can]
  found 4 entries for Test_002
    2024-05-14 : Food 23 ltr Caddy [mdi:food]
    2024-05-14 : Plastics & glass 240 ltr wheeled bin [mdi:recycle]
    2024-05-21 : Paper & card 180 ltr wheeled bin [mdi:newspaper]
    2024-05-21 : Residual 180 ltr bin [mdi:trash-can]
  found 5 entries for Test_003
    2024-05-13 : Garden 240 ltr Bin [mdi:leaf]
    2024-05-13 : Plastics & glass 240 ltr wheeled bin [mdi:recycle]
    2024-05-20 : Paper & card 180 ltr wheeled bin [mdi:newspaper]
    2024-05-13 : Food 23 ltr Caddy [mdi:food]
    2024-05-20 : Residual 180 ltr bin [mdi:trash-can]
  found 5 entries for Test_004
    2024-05-14 : Garden 240 ltr Bin [mdi:leaf]
    2024-05-21 : Paper & card 180 ltr wheeled bin [mdi:newspaper]
    2024-05-14 : Plastics & glass 240 ltr wheeled bin [mdi:recycle]
    2024-05-14 : Food 23 ltr Caddy [mdi:food]
    2024-05-21 : Residual 180 ltr bin [mdi:trash-can]
```